tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala

```diff
@@ -37,23 +44,33 @@ class TestQuintEx extends AnyFunSuite {
     val barDef = QuintDef.QuintOpDef(uid, "bar", "def", lambda)
     val appBar = QuintApp(uid, "bar", List(_42))
     val letBarBeLambdaInAppBar = QuintLet(uid, barDef, appBar)
-
-    // Builtin operator application
-    def app(name: String, args: QuintEx*): QuintApp = QuintApp(uid, name, args)
+    val nIsGreaterThanZero = app("igt", name, _0)
+    // A predicate on ints
+    val intIsGreaterThanZero = QuintLambda(uid, List("n"), "def", nIsGreaterThanZero)
+    val intSet = app("Set", _1, _2, _3)
+    val emptyIntSet = app("Set", _1, _2, _3)
```

Comment:
I couldn't find where this is used.
Would it make sense to include a test for the empty `Set` application?

Reply:
Yes! Absolutely. That war an oversight. Thanks!